repository/src/main/java/org/jbpm/process/workitem/repository/service/RepoService.java

```diff
@@ -124,14 +125,31 @@ protected void loadModules() {
     /*
      * Service related operations
      */
-    
-    public void addService(RepoData service) {        
+
+    public void addService(RepoData service) {
```

Comment:
Where is the remove method? Replacing is one thing, but where we can remove a service

Reply:
Now, we can upload the same GAV jar, we can't distinguish. The feature means, if you upload a new time stamp's GAV jar, we will remove all service tasks for old time stamp's GAV jar. BTW, you can try to upload same GAV's jar,you can find the issue.